activity/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Community Activity</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="/static/timeago.js"></script>
+    <script src="/static/charts.js"></script>
```

Comment:
All of the `<script>` tag can be inside the body (below the content) so it doesn't block content.

`updateChart()` is called after the content anyway, so it doesn't matter.